custom/icds_reports/static/js/directives/mobile-datepicker/mobile-datepicker.directive.js

```diff
@@ -0,0 +1,380 @@
+// note: this code is mostly a direct import of https://github.com/Ankitdhir/angularjs-ios-style-datepicker
+// with a few minor modifications to allow for removing the day of month and small UX tweaks
```

Comment:
Important comment. I don't recommend reviewing this file at all and we instead think of it as a library which - for annoying reasons - has to be included in our source and slightly modified.